agent/agent-core/src/main/java/org/bithon/agent/configuration/ConfigurationManager.java

```diff
@@ -124,7 +124,7 @@ static ConfigurationManager create(File defaultConfigLocation) {
         return new ConfigurationManager(PropertySource.from(PropertySourceType.INTERNAL, defaultConfigLocation, true),
                                         ExternalSource.build(),
                                         CommandLineArgsSource.build("bithon."),
-                                        EnvironmentSource.build("bithon_"));
+                                        EnvironmentSource.build("BITHON_"));
```

Comment:
We can keep the lower case for compatibility.

Reply:
I agree that the case of environment variables should not be strictly enforced. It is only necessary to ensure that the environment variables can be recognized, regardless of whether they are in uppercase or lowercase